### PR TITLE
CPLP-469 - invite business partners

### PR DESCRIPTION
**Pull request by oyo**
- added a page in the user menu "Invite Business Partner"
- copied the logic from existing portal
- new style for dialog and improved input validation

### CODE REVIEW

File: portal/code/cx-portal/src/components/pages/InviteBusinessPartner/index.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import { useState } from 'react'
+import UserService from 'services/UserService'
+import './InviteBusinessPartner.scss'
+
+export default function InviteBusinessPartner() {
+  const [processing, setProcessing] = useState<string>('input')
+
+  const doSubmitInvite = (data: InviteFormData) => {
+    setProcessing('busy')
+    fetch(
```
Comment (fatihayyildiz) [inline portal/code/cx-portal/src/components/pages/InviteBusinessPartner/index.tsx]: why not using httpClient or any axios instance ? this implementation looks bit different than usual way inside the project

Comment (oyo) [reply portal/code/cx-portal/src/components/pages/InviteBusinessPartner/index.tsx]: I copied the logic from the existing portal and only changed it slightly - I will switch to your Axios/HttpClient implementation.

File: portal/code/cx-portal/src/components/pages/InviteBusinessPartner/index.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+import { InviteData } from 'types/userAdministration/UserAdministrationTypes'
+import './InviteBusinessPartner.scss'
+
+export default function InviteBusinessPartner() {
+  const [processing, setProcessing] = useState<string>('input')
+
+  const doSubmitInvite = (data: InviteData) => {
+    setProcessing('busy')
+
+    new UserAdministrationApi(UserService.getToken())
```
Comment (oyo) [inline portal/code/cx-portal/src/components/pages/InviteBusinessPartner/index.tsx:15]: now using a HttpClient based UserAdministrationApi

**Review by fatihayyildiz — APPROVED** [review]
Looks good imo.

**Pull request merged**